squad-get-log-lines

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import sys
+from pathlib import Path
+
+from squad_client.core.api import SquadApi
+from squad_client.core.models import ALL, Build, Squad, Test
+
+squad_host_url = "https://qa-reports.linaro.org/"
+SquadApi.configure(cache=3600, url=os.getenv("SQUAD_HOST", squad_host_url))
+
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="Script to scrape the log parameter from tests in specified projects."
+    )
+
+    parser.add_argument(
+        "--group",
+        required=True,
+        help="The name of the SQUAD group.",
+    )
+
+    parser.add_argument(
+        "--projects",
+        required=True,
+        action="extend",
+        nargs="+",
+        help="A list of SQUAD projects to fetch logs from.",
+    )
+
+    parser.add_argument(
+        "--suites",
+        required=True,
+        action="extend",
+        nargs="+",
+        help="A list of SQUAD test suite names to get logs for.",
+    )
+
+    parser.add_argument(
+        "--debug",
+        required=False,
+        action="store_true",
+        default=False,
+        help="Display debug messages.",
+    )
+
+    parser.add_argument(
+        "--count",
+        required=False,
+        default=2,
+        type=int,
+        help="The number of builds to fetch when fetching logs.",
+    )
+
+    return parser.parse_args()
+
+
+def run():
+    args = parse_args()
+
+    if args.debug:
+        logger.setLevel(level=logging.DEBUG)
+
+    group_name = args.group
+    project_names = args.projects
+
+    group = Squad().group(group_name)
+    for project_name in project_names:
+        project = group.project(project_name)
+        builds = project.builds(count=args.count)
+
+        for build_id in builds:
+            build = Build(build_id)
```

Comment:
Doing `Build(build_id)` makes a new request to SQUAD, sorry I should've documented that somewhere :/ 

What you can do in the line above is
```python
for build_id, build in builds.items():
```

then you wouldn't need `build = Build(build_id)`

Reply:
That's a great spot - thank you for the feedback! I've made this change in both places :)